packages/test/src/workflows/activities-from-class.ts

```diff
@@ -0,0 +1,15 @@
+// @@@SNIPSTART nodejs-activity-from-class-workflow
+import { createActivityHandle } from '@temporalio/workflow';
+import type { ActivitiesWithDependencies } from '../activities/with-deps';
+import { Returner } from '../interfaces';
+
+const { greet } = createActivityHandle<ActivitiesWithDependencies>({
```

Comment:
Is there a way to set `greetString` with the `createActivityHandle()` syntax?

Reply:
No, activities must be initialized in the Worker.